tests/gtests/test_SW_VegProd.cc

```diff
@@ -149,14 +146,11 @@ namespace {
     for (i = 0; i < 12; i++) {
       EXPECT_DOUBLE_EQ(biom2[i], biom1[i] * x);
     }
-
-    // Reset to previous global state
-    Reset_SOILWAT2_after_UnitTest();
   }
 
 
   // Test summing values across vegetation types
-  TEST(VegTest, Summing) {
+  TEST_F(AllTest, Summing) {

```

Comment:
Please change `TEST_F(VegTest, Summing)` to something like `TEST_F(AllTest, VegProdSumming)`